### DISCUSSION TITLE
Remove American style fractions

### DISCUSSION THREAD

**Discussion by ollyde** [tailwindlabs/tailwindcss#13595] [Ideas]
Example feature

https://tailwindcss.com/docs/aspect-ratio

aspect-[4/3]

When doing calculation, for example aspect-[{gridItemAspectRatio}] << 1.333

16/10 for example is 1.6, much easier to work with, instead of mapping everything to some string fraction.

**Comment by adamwathan** [comment]
Can you explain more clearly the actual change you want to see here? Are you saying you want a `aspect-[1.6]` class instead of an `aspect-[16/10]` class?

We just pass whatever value you use to the underlying CSS property, we don't really enforce any opinions here:

https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio
